bot/exts/info/reddit.py

```diff
@@ -188,7 +189,8 @@ async def get_top_posts(self, subreddit: Subreddit, time: str = "all", amount: i
             comments = data["num_comments"]
             author = data["author"]
 
-            title = textwrap.shorten(data["title"], width=64, placeholder="...")
+            title = unescape(data["title"])
+            title = textwrap.shorten(title, width=64, placeholder="...")
```

Comment:
Could we have this and the line above in a single assignment?
```suggestion
            title = textwrap.shorten(
                unescape(data["title"]),
                width=64,
                placeholder="..."
            )
```